packages/client-common/__tests__/integration/insert_specific_columns.test.ts

```diff
@@ -0,0 +1,387 @@
+import { type ClickHouseClient } from '@clickhouse/client-common'
+import { createTableWithFields } from '@test/fixtures/table_with_fields'
+import { createTestClient } from '../utils'
+
+describe('Insert with specific columns', () => {
```

Comment:
could you add examples with nested data as it' done in https://github.com/ClickHouse/clickhouse-connect/blob/526cc8b12a23b36ac1f846180d39fe1c04b263d7/examples/insert_examples.py

Reply:
I need to finally have a look at what's up with Nested via HTTP: https://github.com/ClickHouse/clickhouse-js/issues/89